test/e2e/e2eslow/backup_restore_test.go

```diff
@@ -81,6 +82,49 @@ func TestBackupAndRestore(t *testing.T) {
 	testEtcdRestoreOperatorForS3Source(t, clusterName, s3Path)
 }
 
+// TestBackupAndRestoreABS runs the backup test first, and only runs the restore test after if the backup test succeeds and sets the ABS path
+func TestBackupAndRestoreABS(t *testing.T) {
```

Comment:
Most of the code is duplicate to TestBackupAndRestoreS3().
Can you refactor it to reduce duplication?